README.md

```diff
@@ -1 +1,3 @@
-# Code-review
\ No newline at end of file
+# Code-review
+
+Raad Me
```

Comment:
Please can you add a more in-depth ReadMe file to your project?

You can check https://github.com/microverseinc/readme-template, for a better understanding of how to set up a ReadME file.